From: Steve Best <sbest@redhat.com>
Date: Tue, 2 Nov 2010 19:01:12 -0400
Subject: [scsi] scsi_dh_alua: remove IBM Power Virtual SCSI ALUA
Message-id: <20101102185625.24581.78998.sendpatchset@squad5-lp1.lab.bos.redhat.com>
Patchwork-id: 29006
O-Subject: [PATCH RHEL5.6 BZ567292 V2] scsi_dh_alua: Add IBM Power Virtual SCSI
	ALUA device to dev list
Bugzilla: 567292

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=567292

Description:
------------
Adds IBM Power Virtual SCSI ALUA devices to the ALUA device handler.
this was added to kernel-2.6.18-206.el5.

-V2 removes this patch as VIOS support for this feature has moved out and
this in no longer needed for RHEL 5.6

RHEL Version Found:
-------------------
RHEL 5.5

kABI Status:
------------
No symbols were harmed.

Brew:
-----
http://brewweb.devel.redhat.com/brew/taskinfo?taskID=2867481

Upstream Status:
----------------
back port from

http://git.kernel.org/gitweb.cgi?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=22963a37b3437a25812cc856afa5a84ad4a3f541

Signed-off-by: Brian King <brking@linux.vnet.ibm.com>
Signed-off-by: James Bottomley <James.Bottomley@suse.de>

Test Status:
------------
Tested by me.

===============================================================
Steve Best
IBM on-site partner

Proposed Patch:

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/device_handler/scsi_dh_alua.c b/drivers/scsi/device_handler/scsi_dh_alua.c
index 05a012e..095a058 100644
--- a/drivers/scsi/device_handler/scsi_dh_alua.c
+++ b/drivers/scsi/device_handler/scsi_dh_alua.c
@@ -704,7 +704,6 @@ const struct scsi_dh_devlist alua_dev_list[] = {
 	{"DGC", "DISK"},
 	{"DGC", "VRAID"},
 	{"NETAPP", "LUN"},
-       {"AIX", "NVDISK"},
 	{NULL, NULL}
 };
 
